Resources/Prototypes/DeltaV/Entities/Objects/Specific/Robotics/borg_modules.yml

```diff
@@ -24,33 +24,33 @@
     items:
     - WoodBaton
     - SecurityWhistle
-
-- type: entity
-  id: BorgModuleSecurityPatrol # Peacekeeper Set, Available on Round Start
-  parent: [ BaseBorgModuleSecurity, BaseProviderBorgModule ]
-  name: patrol cyborg module
-  components:
-  - type: Sprite
-    layers:
-    - state: security
-    - state: icon-patrol
-  - type: ItemBorgModule
-    items:
     - FlashlightSecliteRecharging
-    - SecurityWhistle
+# - type: entity 
```

Comment:
dont comment out things just remove them

Reply:
Planning to repurpose the bastion to be fair, hence why I just commented it out

Reply:
Buit I can zap the other one